src/learning/ProfileInitializer.cpp

```diff
@@ -95,10 +95,11 @@ std::vector<Perf> ProfileInitializer::getProfilePerformanceCandidates(
     catAbove = catBelow + 1;
   }
 
+  std::unordered_map<std::string, Category> assig = altPerformance_.getAlternativesAssignments();
```

Comment:
If I remember correctly, the issue comes from WeightedProbability, line 142 & 149